benchmarks/pom.xml

```diff
@@ -151,8 +151,9 @@
       <artifactId>jsr305</artifactId>
     </dependency>
     <dependency>
-      <groupId>com.yahoo.datasketches</groupId>
-      <artifactId>sketches-core</artifactId>
+      <groupId>org.apache.datasketches</groupId>
+      <artifactId>datasketches-java</artifactId>
+      <version>1.1.0-incubating</version>
```

Comment:
How about defining the version in the root POM (to ensure the value here is consistent with the one in extensions-core/datasketches/pom.xml)?